package.json

```diff
@@ -117,6 +105,12 @@
       "limit": "10 KB"
     }
   ],
+  "files": [
```

Comment:
I'm curious if we really need the files property (will have to double-check on this)

Reply:
As per package.json docs: `The optional files field is an array of file patterns that describes the entries to be included when your package is installed as a dependency. File patterns follow a similar syntax to .gitignore, but reversed: including a file, directory, or glob pattern (*, **/*, and such) will make it so that file is included in the tarball when it's packed. Omitting the field will make it default to ["*"], which means it will include all files.`

I know we have a npmignore for this, but I think this a useful property